mlflow/projects/_project_spec.py

```diff
@@ -1,25 +1,44 @@
 """Internal utilities for parsing MLproject YAML files."""
 
 import os
+import yaml
 
 from six.moves import shlex_quote
 
 from mlflow import data
+from mlflow.utils.exception import ExecutionException
+
+
+def load_project(directory):
+    mlproject_path = os.path.join(directory, "MLproject")
```

Comment:
Let's make environment variables with MLproject and conda.yaml, as these are important default constants.